exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebUtils.java

```diff
@@ -55,7 +55,10 @@
   public static String getCsrfTokenFromHttpRequest(HttpServletRequest request) {
     // No need to create a session if not present (i.e. if a user is logged in)
     HttpSession session = request.getSession(false);
-    return session == null ? "" : (String) session.getAttribute(WebServerConstants.CSRF_TOKEN);
+    String res = session == null ? "" :
+      (String) session.getAttribute(WebServerConstants.CSRF_TOKEN);
+    // In case session is created when authentication is disabled
+    return res == null ? "" : res;
```

Comment:
What happens without these changes? Is it possible to revert these if no effect for anything?

Reply:
if drill.exec.security.user.auth.enabled=false, WebServerConstants.CSRF_TOKEN attribute will not be added into the session, after I created the session, this function will return null, which will cause exception when rending the profile page